pkg/process/status/expvars.go

```diff
@@ -272,5 +275,10 @@ func InitInfo(hostname string, processModuleEnabled bool, eps []apicfg.Endpoint)
 		expvar.Publish("endpoints", expvar.Func(publishEndpoints(eps)))
 		expvar.Publish("drop_check_payloads", expvar.Func(publishDropCheckPayloads))
 		expvar.Publish("system_probe_process_module_enabled", publishBool(processModuleEnabled))
+
+		// Run a profile & telemetry server.
+		if config.GetBool("telemetry.enabled") {
+			http.Handle("/telemetry", telemetry.Handler())
```

Comment:
Just for my understanding, what's the difference between this endpoint and `/debug/vars`?

Reply:
Ah! `/telemetry` was something that was added along with workloadmeta to expose prometheus metrics. Since it used the same mux as the expvar server, and didn't have any tests - I decided to wrap it in the expvar component and add a test for it.